### DISCUSSION TITLE
Formatting warning issue

### DISCUSSION THREAD

**Discussion by 0xblur** [LazyVim/LazyVim#1063] [Q&A]
I didn't know what title to put it because I really don't know what is happening. I was importing the autoformat config from `nvim-lspconfig` manually, and everything works, but I get this error. I don't know what to do because everything seems OK.

```Lua
Invalid plugin spec {
  enabled = <function 1>,
  format = <function 2>,
  get_formatters = <function 3>,
  notify = <function 4>,
  setup = <function 5>,
  supports_format = <function 6>,
  toggle = <function 7>
}
Invalid plugin spec {
  diagnostic_goto = <function 1>,
  get = <function 2>,
  has = <function 3>,
  on_attach = <function 4>,
  resolve = <function 5>
}
```

**Comment by folke** [comment]
What do you mean? You should not import that anywher

**Reply by 0xblur** [reply]
Instead of:
```Lua
spec = { 
  { LazyVim/LazyVim, import = "lazyvim.plugins" }
```
I have copied the `lua/lazyvim` folder in order to build this configuration from scratch and familiarize myself with the plugins, options, and keymaps during the building process. I have also deleted whatever I didn't find useful to me. I had added everything progressively without much issue, but after I added the LSP formatting configuration and its respective auto-commands from `lazyvim.util`, I started getting that error from above.

**Reply by folke** [reply]
Sorry, but I don't support using LazyVim this way. You're on your own
